### ISSUE TITLE
[Bug]: does not work with v10

### ISSUE THREAD

**Issue by zarmstrong** [NateHawk85/module-profiles#20]
### Describe the Bug

Does not work with Foundry V10

### Steps to Replicate

```bash
1) Install and enable module
```

### Desired Outcome

Works as usual

### Actual Outcome

ERrors in console:

```
foundry.js:711 TypeError: object is not iterable (cannot read property Symbol(Symbol.iterator))
[Detected 1 package: module-profiles]
    at settings.ts:18:1
    at Array.map (<anonymous>)
    at Module.getCurrentModuleConfiguration (settings.ts:18:1)
    at buildDefaultProfile (settings-utils.ts:26:42)
    at Module.registerSettings (settings-utils.ts:15:1)
    at Object.registerModuleSettings [as fn] (settings.ts:6:5)
    at #call (foundry.js:688:20)
    at Hooks.callAll (foundry.js:645:17)
    at Game.setupGame (foundry.js:7826:11)
    at async Game._initializeGameView (foundry.js:8877:5)
    at async Game.initialize (foundry.js:7687:5)
```

### Foundry Version

v10 b285

### System and Version

Pathfinder 2e 4.0.3

### Module Profiles Version

1.0.5

### What browser did you see this on?

Chrome

### What OS were you using?

Win 10

### Installed Modules + Additional Info

- Module Profiles v1.0.5

**Comment by NateHawk85** [comment]
Should be fixed in v1.0.6!

**Comment by zarmstrong** [comment]
Thanks Nate!
